### ISSUE TITLE
table row announced as "clickable" even though the @click is null and the row is not clickable.

### ISSUE THREAD

**Issue by mhabbasi-gh** [nvaccess/nvda#10836]
in Vuejs written code, NVDA reads a table row as "clickable row" even if the row is not clickable. I believe NVDA announces "clickable" if "@click" exists even if it is null. NVDA should not announce "clickable" when click event is equal to null like "@click=''".

I am using NVDA 2019.2.1.  Chrome browser. Windows 10.

**Comment by DrSooom** [comment]
@mohammad-bede: Please specify the Google Chrome version. thanks.

**Comment by mohammad-bede** [comment]
@DrSooom My chrome is Version 80.0.3987.122 (Official Build) (64-bit)

**Comment by Adriani90** [comment]
@mohammad-bede please provide a test example. From what you describe I cannot reproduce in Chrome 81 with NVDA 2020.1.
Can you confirm?

**Comment by Adriani90** [comment]
@mhabbasi-gh are you still having this issue with NVDA 2023.1 in Chrome or Edge 112?

**Comment by Adriani90** [comment]
We didn't get any updates from the initial author since many years, I am closing as abandoned. If you are still having this issue, please comment and we can reopen.
